app/pages/lab-pages-editor/components/TasksPage/TasksPage.jsx

```diff
@@ -182,16 +195,26 @@ export default function TasksPage() {
 
     update({ tasks: newTasks });
   }
+  
+  // Limited Branching Rule:
+  // 0. a Step can only have 1 branching task (single answer question task)
+  // 1. if a Step has a branching task, it can't have any other tasks.
+  // 2. if a Step already has at least one task, any added question task must be a multiple answer question task.
+  // 3. if a Step already has many tasks, any multiple answer question task can't be transformed into a single answer question task. 
```

Comment:
This description is different than the solution described in Challenge 3.5 in the [notes doc](https://docs.google.com/document/d/1YVZpTkH7FkDXphn1NUg8E1tOElQdLMnJnznsqyLjANw/edit).

> disallow a single-answer question task to be combined with another single-answer question task, not disallowing the possibility of combining it with all task types. 

Do you plan to change `enforceLimitedBranchingRule` after this PR? Before or after internal testing?

Reply:
That's correct, the Limited Branching Rule in this PR is the _old_ version. The new version will be implemented in a future PR, with a target to be done before internal testing starts.

[(Slack ToDo list, 13 May)](https://zooniverse.slack.com/archives/C39AUB9UK/p1715631331033939)